<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceSettings</class>
 <widget class="QWidget" name="AppearanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string comment="@title:group Group box label">General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QCheckBox" name="kcfg_TerminalHighlightOnManualActivation">
        <property name="whatsThis">
         <string comment="@info:whatsthis">Enabling this option will make Yakuake briefly display a colored overlay above the newly focused terminal when moving focus between multiple terminals in a session, as well as over the currently focused terminal when switching to a session with multiple terminals.</string>
        </property>
        <property name="text">
         <string comment="@option:check">Highlight terminals when moving focus between them</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_HideSkinBorders">
        <property name="text">
         <string>Hide borders</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string comment="@title:group">Window Background</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="backgroundColorLabel">
          <property name="text">
           <string comment="@label:chooser">Background color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="kcfg_BackgroundColor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>41</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_Translucency">
        <property name="whatsThis">
         <string comment="@info:whatsthis">Translucency requires support by the selected skin as well as your desktop environment. The default skin supports translucency.</string>
        </property>
        <property name="text">
         <string comment="@option:check">Use skin translucency if supported</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>7</number>
        </property>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="backgroundColorOpacityLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="@label:spinbox">Background color opacity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_BackgroundColorOpacity">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string comment="@info:whatsthis">When using translucency, the background color will be filled in using the given opacity before the skin elements are painted on top of the background. This allows influencing the opacity and color tint of translucent areas of the skin.</string>
          </property>
          <property name="suffix">
           <string comment="@label Spinbox unit suffix">%</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>64</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string comment="@title:group">Skin</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="0" colspan="2">
       <widget class="KNSWidgets::Button" name="ghnsButton">
        <property name="text">
         <string comment="@action:button">Get New Skins...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="removeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="@action:button">Remove Skin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="installButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string comment="@action:button">Install Skin...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="kcfg_SkinInstalledWithKns">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="kcfg_Skin">
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListView" name="skinList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_TerminalHighlightOnManualActivation</tabstop>
  <tabstop>kcfg_BackgroundColor</tabstop>
  <tabstop>kcfg_Translucency</tabstop>
  <tabstop>kcfg_BackgroundColorOpacity</tabstop>
  <tabstop>skinList</tabstop>
  <tabstop>installButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>ghnsButton</tabstop>
  <tabstop>kcfg_Skin</tabstop>
  <tabstop>kcfg_SkinInstalledWithKns</tabstop>
 </tabstops>
 <customwidgets>
  <customwidget>
   <class>KNSWidgets::Button</class>
   <extends>QPushButton</extends>
   <header>KNSWidgets/Button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_Translucency</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_BackgroundColorOpacity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_Translucency</sender>
   <signal>toggled(bool)</signal>
   <receiver>backgroundColorOpacityLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
